wallet/wallet.c

```diff
@@ -549,6 +606,27 @@ u8 *derive_redeem_scriptsig(const tal_t *ctx, struct wallet *w, u32 keyindex)
 	return bitcoin_scriptsig_p2sh_p2wpkh(ctx, &key);
 }
 
+void wallet_compute_max(struct wallet *w, u32 max_utxos, struct amount_sat *sat)
+{
+	size_t i = 0;
+	struct utxo **available;
+
+	available = sorted_confirmed_utxos(NULL, w, output_state_available);
+
+	*sat = AMOUNT_SAT(0);
+	/* Mark them all as reserved and sum up total */
```

Comment:
I can't see us marking them as reserved here?

Reply:
this comment was made in error, it's been removed. :D